tests/integrations/tests/test_google.py

```
@@ -887,7 +887,8 @@ def test_25_speech_generation_language_support(self, google_client, test_config,
             assert_valid_speech_response(wav_audio, expected_audio_size_min=1000)
 
     @skip_if_no_api_key("google")
-    def test_26_extended_thinking(self, google_client, test_config):
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("thinking"))
+    def test_26_extended_thinking(self, google_client, test_config, provider, model):
```

Ruff (0.14.8)

Unused method argument: `test_config`

(ARG002)